config/initializers/lock_manager.rb

```diff
@@ -19,7 +19,7 @@ def self.with_transaction_lock(lock_name, options={})
     lock_id = Zlib.crc32(lock_name.to_s)
     ActiveRecord::Base.transaction do
       lock.retry_with_timeout do 
-        if (ActiveRecord::Base.connection.execute("SELECT pg_try_advisory_xact_lock(#{lock_id})").values[0][0] == 't')
+        if (ActiveRecord::Base.connection.execute("SELECT pg_try_advisory_xact_lock(#{lock_id})").values[0][0])
```

Comment:
In rails 5, `ActiveRecord::Base.connection.execute("SELECT pg_try_advisory_xact_lock(#{lock_id})").values[0][0]` is `true` not the string `'t'`. This corrects that.

Reply:
What was the original need for this LockManager? Feels like so much (all) of this is handled natively already. Could we rip out the lock manager entirely?

Reply:
It's not exactly handled. The problem is that I needed way to prevent modifications to a set of models that didn't cause a database lock itself. I don't remember the exact reason but it had to do with Stripe's webhooks and it was causing a deadlock.

I don't have the exact reason top of mine and I have no doubt I'd make different decisions now but I know a number of things to avoid this and this was the best I came up with.